cumulus/parachains/runtimes/assets/asset-hub-rococo/tests/tests.rs

```diff
@@ -683,6 +683,7 @@ mod asset_hub_rococo_tests {
 			bridging_to_asset_hub_wococo,
 			WeightLimit::Unlimited,
 			Some(xcm_config::bridging::XcmBridgeHubRouterFeeAssetId::get()),
+			Some(xcm_config::TreasuryAccount::get().unwrap()),
```

Comment:
TreasuryAccount is an `Option` right now. I think we could simply use `xcm_config::TreasuryAccount::get()`

Reply:
yes, I know, but I wanted to be explicit that this test uses `Some`